### DISCUSSION TITLE
Argument Error: no such member :billing_mode

### DISCUSSION THREAD

**Discussion by steverhoades** [Dynamoid/dynamoid#488] [Q&A]
Out of curiosity has anyone seen this error before? I cannot seem to work past this. :billing_mode is in the create_table.rb class.

Using Ruby 5.3, Dynamoid v3.6.0. Seeing this error while attempting to run RSPEC tests.

Using:
```
Rspec.configure do |config|
  config.before do
    Aws.configure[:stub_responses] = true
  end
end
```

The error occurs when attempting to save the model.
```
pet = Pet.new(name: "pete")
pet.save # <--- error here
```

**Comment by andrykonchin** [comment]
I haven't heard about issues with stubbing test responses in AWS. But there is a fresh enough issue which may be similar to this one - https://github.com/Dynamoid/dynamoid/issues/486.

Could you please place an issue and provide example of test to reproduce it?
